pages/api/FolderApi.tsx

```diff
@@ -0,0 +1,17 @@
+import getDate from "./GetData";
+
+export async function getFolderUserData() {
+  return getDate("/users/1");
+}
+
+export async function getFoldersData() {
```

Comment:
마찬가지로 api endpoint가 말하는 리소스 위치 처럼, 가져오려는 리소스(데이터)를 설명할 수 있게 수정해도 좋겠습니다
```suggestion
export async function getUserFoldersData(userId = 1) {
```